swix/res/Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest.swr

```diff
@@ -0,0 +1,14 @@
+use vs
+
+package name=Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest
+        version=$(Version)
+        vs.package.type=vsix
+        vs.package.installSize=22265856
```

Comment:
Ditto; but also should this be duplicated size? (We can fix it later)

Reply:
Not currently used, will fix later.